### PR TITLE
feat: literally just getting it to run

### PR DESCRIPTION
**Pull request by estrattonbailey**
That's it. That's the commit. All tests pass.

Screen Shot 2023-05-02 at 4 19 34 PM

### CODE REVIEW

File: src/walletConnect/types.ts
Comment (estrattonbailey) [inline src/walletConnect/types.ts:1]: Just moved these over and updated imports

File: src/walletConnect/index.tsx
Comment (estrattonbailey) [inline src/walletConnect/index.tsx:1]: Only updated imports and move the types in this file, no functional changes.

File: src/walletConnect/__tests__/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+jest.mock('@/handlers/web3');
+jest.mock('@/screens/Portal');
+jest.mock('@/walletConnect/sheets/AuthRequest', () => ({
+  AuthRequest: jest.fn(),
+}));
+
+// import { parseRPCParams } from '@/walletConnect';
+// import { RPCMethod } from '@/walletConnect/types';
+
+test(`works`, () => {});
```
Comment (estrattonbailey) [inline src/walletConnect/__tests__/index.test.ts:25]: All for this

File: src/handlers/__tests__/deeplinks.test.ts [hunk 2]
```diff
@@ -19,11 +19,7 @@ import {
 import { fetchReverseRecordWithRetry } from '@/utils/profileUtils';
 
 jest.mock('@/redux/store');
-jest.mock('@/redux/walletconnect', () => ({
-  walletConnectOnSessionRequest: jest.fn(),
-  walletConnectRemovePendingRedirect: jest.fn(),
-  walletConnectSetPendingRedirect: jest.fn(),
-}));
+jest.mock('@/redux/walletconnect');
```
Comment (estrattonbailey) [inline src/handlers/__tests__/deeplinks.test.ts:22]: Moved mocks adjacent to these files, so no need to write the mock implementations here anymore.

File: src/walletConnect/sheets/AuthRequest.tsx [hunk 3]
```diff
@@ -10,7 +10,6 @@ import {
   AccentColorProvider,
 } from '@/design-system';
 import ButtonPressAnimation from '@/components/animations/ButtonPressAnimation';
-import useAccountProfile from '@/hooks/useAccountProfile';
```
Comment (estrattonbailey) [inline src/walletConnect/sheets/AuthRequest.tsx:13]: Wasn't being used

**Review by benisgold — APPROVED** [review]

**Pull request merged**